lib/adminlte-rails/source_file.rb

```diff
@@ -62,30 +49,17 @@ def select(msg, elements)
   end
 
   def fetch_javascripts(remote, tag)
-    js_path = "#{remote}/raw/#{tag}/js"
-    get "#{js_path}/bootstrap.min.js", 'javascripts/bootstrap.min.js'
-    get "#{js_path}/AdminLTE/app.js", 'javascripts/admin-lte.js'
-  end
-
-  def fetch_fonts(remote, tag)
-    fonts_path = "#{remote}/raw/#{tag}/fonts"
-    get "#{fonts_path}/FontAwesome.otf", 'fonts/FontAwesome.otf'
-    get "#{fonts_path}/fontawesome-webfont.eot", 'fonts/fontawesome-webfont.eot'
-    get "#{fonts_path}/fontawesome-webfont.svg", 'fonts/fontawesome-webfont.svg'
-    get "#{fonts_path}/fontawesome-webfont.ttf", 'fonts/fontawesome-webfont.ttf'
-    get "#{fonts_path}/fontawesome-webfont.woff", 'fonts/fontawesome-webfont.woff'
-    get "#{fonts_path}/glyphicons-halflings-regular.eot", 'fonts/glyphicons-halflings-regular.eot'
-    get "#{fonts_path}/glyphicons-halflings-regular.svg", 'fonts/glyphicons-halflings-regular.svg'
-    get "#{fonts_path}/glyphicons-halflings-regular.ttf", 'fonts/glyphicons-halflings-regular.ttf'
-    get "#{fonts_path}/glyphicons-halflings-regular.woff", 'fonts/glyphicons-halflings-regular.woff'
-    get "#{fonts_path}/ionicons.eot", 'fonts/ionicons.eot'
-    get "#{fonts_path}/ionicons.svg", 'fonts/ionicons.svg'
-    get "#{fonts_path}/ionicons.ttf", 'fonts/ionicons.ttf'
-    get "#{fonts_path}/ionicons.woff", 'fonts/ionicons.woff'
+    js_path = "#{remote}/raw/#{tag}/dist/js"
+    
+    get "#{js_path}/app.js", 'javascripts/admin-lte.js'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.